src/index.html

```diff
@@ -1,12 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--start-day-sun"></div>
```

Comment:
all modifier classes should be on the calendar block

```suggestion
    <div class="calendar calendar--start-day-sun">
      <div class="calendar__day"></div>
```

Reply:
also you haven't completed this task

> Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for)